dataactvalidator/filestreaming/csvAbstractReader.py

```diff
@@ -67,25 +66,54 @@ def openFile(self,region,bucket,filename,csvSchema,bucketName,errorFilename):
             raise ResponseException("Error in header row: CSV file must use only '|' or ',' as the delimiter", StatusCode.CLIENT_ERROR, ValueError, ValidationError.headerError)
 
         self.delimiter = "|" if line.count("|") != 0 else ","
-        for row in csv.reader([line],dialect='excel', delimiter=self.delimiter):
-            for cell in row :
-                headerValue = FieldCleaner.cleanString(cell)
-                if( not headerValue in possibleFields) :
+
+        # check to see if header contains long or short column names
+        # TODO: is there a better way to do this?
+        validation_db = ValidatorValidationInterface()
+        longNameDict = validation_db.getLongToShortColname()
+        headerNames = line.split(self.delimiter)
```

Comment:
The deleted line 70 above, or line 91 below are examples of how we have been reading csv lines in other places.  By splitting "line" without reading it through csv.reader, we bypass the handling of excel dialect, possible quoting of fields, and whatever else the csv package checks for.  So ' "header 1, header 2", header 3' would be interpreted by csv.reader as 2 elements ("header 1, header 2") and (header 3), but going through split will give 3 elements ("header 1), ( header 2"), and (header 3).